using-kite-properties.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: Using Kite Properties
+---
+
+Kite gives you the option of changing settings of existing system properties to enhance performance. You also have the option of creating custom properties.
+
+## Setting Kite Properties
+
+There are several dataset properties that affect how files are written. These settings are especially relevant for Parquet because it buffers records in memory. 
+
+You can set properties on your datasets with [`DatasetDescriptor.Builder.property`][dataset-descriptor-builder] when using the API, or with the `--set` option from the command line.
+
+### kite.write.cache-size
+
+`kite.write.cache-size` controls the number of files kept open by an HDFS or Hive dataset writer.
+
+Writers open one file per partition to which they write records. When the writer receives a record that goes in a new partition (one for which there isn't an open file) it creates a new file in that partition. If the number of open files exceeds the cache size, Kite closes the file that was used least recently.
+
+### parquet.block.size
+
+Kite passes descriptor properties to the underlying file formats. For example, Parquet defines `parquet.block.size`, which is approximately the amount of data that is buffered before writing a group of records (a "row group"). `parquet.block.size` defaults to 128MB.
+
+#### Avoiding Parquet OutOfMemory Exceptions
+
+The amount of data kept in memory for each file could be up to the Parquet block size in bytes. That means that the upper bound for a writer's memory consumption is `parquet.block.size` * `kite.writer.cache-size`. It is important that this number doesn't exceed a reasonable portion of the heap memory allocated to the process, or else the write could fail with an `OutOfMemoryException`. 
+
+```
+kite-dataset update <uri> --set kite.writer.cache-size=2
+```
+
+Note that Cloudera does not recommend decreasing the `parquet.block.size`.
```

Comment:
Yes, we don't need to discuss how or why you might change the Parquet block size. Just say that the recommended way to avoid OOM is to write to fewer files and that the `CrunchDatasets.partition` methods are intended to help with that.